src/box/lua/config/init.lua

```diff
@@ -356,6 +356,11 @@ function methods._startup(self, instance_name, config_file)
     self:_set_status_based_on_alerts()
 end
 
+function methods._print_env_list(self)
+    local env_source = require('internal.config.source.env').new()
+    io.stdout:write(env_source:_env_list())
```

Comment:
It would be logical, but this way I would require `tabulate` in two places:

* In the env source to use `tabulate.SPACER`.
* Here to encode the data.

So, I placed the encoding right into the env source.

We still can split data generation and data encoding, but it would involve some intermediate format that looks too complex for the simple task.

What do you think?

Reply:
As we've already discussed, I have no strong opinion regarding any of these approaches. So, feel free to leave it as is.